test/global_plugins.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2012, Willow Garage, Inc.
```

Comment:
review the copyright

Reply:
Added a second copyright line and line about what was modified at the bottom of the license.